source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewController.mm

```diff
@@ -45,6 +64,11 @@ - (instancetype)init:(ACOAdaptiveCard *)card
             _hostConfig = [config getHostConfig];
         }
         _guideFrame = frame;
+        _imageViewMap = [[NSMutableDictionary alloc] init];
+        _textMap = [[NSMutableDictionary alloc] init];
+        _serial_queue = dispatch_queue_create("io.adaptiveCards.serial_queue", NULL);
+        _serial_text_queue = dispatch_queue_create("io.adaptiveCards.serial_text_queue", DISPATCH_QUEUE_SERIAL);
+        _serialNumber = 0;
```

Comment:
You don't have to initialize these here. They've already been initialized in `initWithNibName:bundle:`